### PR TITLE
fix: add Devanshukoli

### LINKED ISSUE #1667: [PROFILE ADDITION] <Devanshu Koli>
### Profile screenshot

My GitHub Profile Image:
git hub profile

### PR DESCRIPTION
**Pull request by Devanshukoli**
resolves #1667 

## Checklist for a profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please revert the changes in the other files and keep your profile changes only.
If you need any help doing that, let us know and we'll gladly help. 🙂

**Comment by Devanshukoli** [comment]
Hey, @Panquesito7 I tried reverting the changes but I used 'git revert HEAD' this is what I used in the terminal and I don't know now what to do? Could you please guide me through it?

**Comment by Panquesito7** [comment]
> I don't know now what to do? Could you please guide me through it?

You can copy all the changes from the [original files](https://github.com/EddieHubCommunity/awesome-github-profiles/) and put them in your fork, then only add your profile. 🙂

**Comment by Devanshukoli** [comment]
@Panquesito7 I have done the changes as you told. So would you kind take a look as well?

File: _layouts/default.html [hunk 1]
```diff
@@ -61,4 +61,4 @@ <h2 class="project-tagline">
       >
     </footer>
   </main>
-</body>
+</body>
```
Comment (Panquesito7) [inline _layouts/default.html:64]:
```suggestion
</body>

```

File: generate.js [hunk 2]
```diff
@@ -39,4 +39,4 @@ fs.readdir(
       }
     );
   }
-);
+);
```
Comment (Panquesito7) [inline generate.js:42]:
```suggestion
);

```

File: package-lock.json [hunk 3]
```diff
@@ -33,4 +33,4 @@
       "dev": true
     }
   }
-}
+}
```
Comment (Panquesito7) [inline package-lock.json:36]:
```suggestion
}

```

File: package.json [hunk 4]
```diff
@@ -22,4 +22,4 @@
   "devDependencies": {
     "prettier": "2.3.2"
   }
-}
+}
```
Comment (Panquesito7) [inline package.json:25]:
```suggestion
}

```

File: profiles/Devanshukoli.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/99028806/213636924-304e3333-6ab8-426f-a416-410604854df9.png",
+    "issueId": 1667,
+    "name": "Devanshu Koli",
+    "username": "Devanshukoli"
+  }
```
Comment (Panquesito7) [inline profiles/Devanshukoli.json:6]:
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Looks like the endlines were modified. Feel free to accept the suggestions by clicking on `Commit suggestion`.

File: profiles/Devanshukoli.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/99028806/213636924-304e3333-6ab8-426f-a416-410604854df9.png",
+    "issueId": 1667,
+    "name": "Devanshu Koli",
+    "username": "Devanshukoli"
```
Comment (Panquesito7) [inline profiles/Devanshukoli.json:5]: Wrong indentation.
```suggestion
  "image": "https://user-images.githubusercontent.com/99028806/213636924-304e3333-6ab8-426f-a416-410604854df9.png",
  "issueId": 1667,
  "name": "Devanshu Koli",
  "username": "Devanshukoli"
```

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Comment by Devanshukoli** [comment]
Thanks for the help, It was a really great experience. @Panquesito7

**Comment by Panquesito7** [comment]
Not sure you want to close this PR? If you close, you won't be able to get your profile added to the list.

**Comment by Devanshukoli** [comment]
Ok, got it.

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻 ⚡

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by Devanshukoli** [comment]
Yeyy! Thanks for guiding me this through @Panquesito7 and for merging @Pradumnasaraf

**Pull request merged**